### PR TITLE
feat: render markdown

### PR DESCRIPTION
**Pull request by safeamiiir**
- Add markdown Library in the frontend to preview `.md` files
- Also make `text/plain` content type readable in the file Viewer.

### CODE REVIEW

**Review by samiralavi — APPROVED** [review]
Thanks @safeamiiir great work and nice feature!

File: aqueductcore/frontend/src/components/organisms/Attachments/Viewer/index.tsx [hunk 1]
```diff
@@ -145,36 +171,46 @@ function Viewer({
...
-            ) : null
+          </SpecialFilePreview>
+          /* MARKDOWN */
+        ) : info?.type === "MARKDOWN" ? (
+          <SpecialFilePreview>
+            {info && <Markdown urlTransform={(url) => mdUrlTransformer(url, experimentId)}>{String(info?.data)}</Markdown>}
+          </SpecialFilePreview>
+          /* TEXT */
+        ) : info?.type === "text/plain" ? (
+          <TextViewer readOnly>{String(info.data)}</TextViewer>
```
Comment (jatinriverlane) [inline aqueductcore/frontend/src/components/organisms/Attachments/Viewer/index.tsx:184]: Can add padding to text viewer

**Review by jatinriverlane — APPROVED** [review]
Works perfectly. Thanks @safeamiiir

File: aqueductcore/frontend/src/components/organisms/Attachments/Viewer/index.tsx (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/components/organisms/Attachments/Viewer/index.tsx:185]: Good point, it's been added now

**Pull request merged**